### PR TITLE
Add a github action to automatically precheck new issues for problems

### PR DESCRIPTION
**Pull request by felix920506**
The source code of the script can be found here: https://github.com/felix920506/jellychan
The script in action can be found here: https://github.com/felix920506/jellyfin/issues/20

The script checks for some common errors in issue reports and comments on them.

**Issues**
None

### CODE REVIEW

File: .github/workflows/issue-template-check.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+          cache: 'pip'
+      - name: install python packages
+        run: pip install -r requirements.txt
+      - name: check and comment issue
+        run: python3 single_issue_gha.py
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (crobibero) [inline .github/workflows/issue-template-check.yml:26]:
```suggestion
          GH_TOKEN: ${{ secrets.JF_BOT_TOKEN }}
```

File: .github/workflows/issue-template-check.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+jobs:
+  check_issue:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: pull in script
+        uses: actions/checkout@v4
+        with:
+          repository: felix920506/jellychan
```
Comment (nielsvanvelzen) [inline .github/workflows/issue-template-check.yml:15]: Should be upstreamed and given a proper name (something like jellyfin-bot-triage?)

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]

File: .github/workflows/issue-template-check.yml (hunk 2 above)
Comment (felix920506) [reply .github/workflows/issue-template-check.yml:15]: In a different repo under the Jellyfin organization?

Comment (felix920506) [reply .github/workflows/issue-template-check.yml]: this has been sorted

**Review by JPVenson — APPROVED** [review]

File: .github/workflows/issue-template-check.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+          repository: jellyfin/jellyfin-triage-script
+      - name: install python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+          cache: 'pip'
+      - name: install python packages
+        run: pip install -r main-repo-triage/requirements.txt
+      - name: check and comment issue
+        run: cd main-repo-triage/; python3 single_issue_gha.py
```
Comment (crobibero) [inline .github/workflows/issue-template-check.yml:24]:
```suggestion
        working-directory: ./main-repo-triage
        run: python3 single_issue_gha.py
```

**Review by crobibero — APPROVED** [review]

**Review by nielsvanvelzen — APPROVED** [review]

**Review by joshuaboniface — APPROVED** [review]

**Pull request merged**